demos/python_demos/single_human_pose_estimation_demo/models.lst

```diff
@@ -0,0 +1,11 @@
+# This file can be used with the --list option of the model downloader.
+mobilenet-ssd
+pedestrian-and-vehicle-detector-adas-0001
```

Comment:
Why the specific numbers here? The demo ought to work with any version of the detector.

@Wovchena I saw that you suggested this, so maybe you could answer. :)

Reply:
There is only one version of `pedestrian-and-vehicle-detector-adas`, so I don't see any difference between `pedestrian-and-vehicle-detector-adas-0001` and `pedestrian-and-vehicle-detector-adas-????` and I find any of the ways of writing acceptable.

Reply:
The wildcards are normally used so that we don't have to update every `models.lst` if a new model in a series comes out.

Reply:
But, different version model can have different format input or/and output. For example, `person-detection-retail-0013` and `person-detection-retail-0002`. And we will have not correct 'models.lst'. Is it ok?

Reply:
It has to be correct, obviously. If the demo doesn't actually work with some of the models in a series, then you should explicitly specify the model numbers. But otherwise you should use wildcards.